frontend/src/components/NotFound.vue

```diff
@@ -1,8 +1,11 @@
 <script setup lang="ts">
 import router from '@/router';
 import { useRoute } from 'vue-router';
+import { showToastErrorTopRight } from '@/store/ToastState';
 
-console.log(`Route "${useRoute().path}" not found, redirect to home.`);
+const notFoundMsg = `Route "${useRoute().path}" not found, redirect to home.`;
+console.log(notFoundMsg);
+showToastErrorTopRight(notFoundMsg);
```

Comment:
when user reaches to a non exists page, pop up the toast error message to them, and redirect back to landing page.